helm-system-packages-brew.el

```diff
@@ -0,0 +1,214 @@
+      (setq str (concat "* Description: " (alist-get 'desc pkg-desc-alist) "\n"
+	      "* Version: "(alist-get 'stable (alist-get 'versions pkg-desc-alist)) "\n"
+	      "* URL: "(alist-get 'homepage pkg-desc-alist) "\n"  "\n"
+	      "* Dependencies:\n" "   " (mapconcat 'identity (alist-get 'dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Optional dependencies:\n" "   "(mapconcat 'identity (alist-get 'optional_dependencies pkg-desc-alist) "\n   ") "\n\n"
+	      "* Options:\n" (mapconcat (lambda (pkg-option)
+					(concat (alist-get 'option pkg-option)  "\n"
+						"    " (alist-get 'description pkg-option)  "\n"))
+				      (alist-get 'options pkg-desc-alist) "\n")
+	      "\n\n"
+	      "* Caveats: " (alist-get 'caveats pkg-desc-alist) "\n"))
+     (add-to-list 'desc-list `(uninstalled (,pkg . ,str)))
+     (setq i (+ i 1)))
+    (helm-system-packages-show-information desc-list)))
+
+(defun helm-system-package-brew-browse-url (_candidate)
+   (let* ((descriptions (json-read-from-string (with-temp-buffer
+						(apply 'call-process "brew" nil t nil "info" "--json=v1" (helm-marked-candidates)) 
+						(buffer-string)))))
+    (helm-system-packages-browse-url (mapcar (lambda(pkg)
+					       (alist-get 'homepage pkg))
+					     descriptions))))
+
+(defun helm-system-package-brew-link-app (_candidate)
+  (helm-system-packages-brew-run  "brew" "link"))
+
+(defun helm-system-package-brew-unlink-app (_candidate)
+  (helm-system-packages-brew-run  "brew" "unlink"))
+
+
+
+(defun helm-system-packages-brew-run (command &rest args)
+  "COMMAND to run over `helm-marked-candidates'.
+
+COMMAND will be run in an Eshell buffer `helm-system-packages-eshell-buffer'."
+  (require 'esh-mode)
+  (let ((arg-list (append args (helm-marked-candidates)))
+        (eshell-buffer-name helm-system-packages-eshell-buffer))
+    ;; Refresh package list after command has completed.
+    (push command arg-list)
+    (eshell)
+    (if (eshell-interactive-process)
+        (message "A process is already running")
+      (add-hook 'eshell-post-command-hook 'helm-system-packages-refresh nil t)
+      (add-hook 'eshell-post-command-hook
+                (lambda () (remove-hook 'eshell-post-command-hook 'helm-system-packages-refresh t))
+                t t)
+      (goto-char (point-max))
+      (insert (mapconcat 'identity arg-list " "))
+      (when helm-system-packages-auto-send-commandline-p
+        (eshell-send-input)))))
+
+(defcustom helm-system-packages-brew-actions
+  '(("Show package(s)" . helm-system-packages-brew-info)
+    ("Install (`C-u' to reinstall)" .
+     (lambda (_)
+       (if helm-current-prefix-arg
+	   (helm-system-packages-brew-run  "brew" "reinstall")
+	 (helm-system-packages-brew-run "brew" "install"))))
+    ("Uninstall (`C-u' to uninstall all versions)" .
+     (lambda (_)
+       (helm-system-packages-brew-run "brew" "uninstall"
+                                         (when helm-current-prefix-arg "--force"))))
+    
```

Comment:
No linebreak here.

Reply:
Should be fixed by https://github.com/tuedachu/helm-system-packages/pull/1.